### PR TITLE
testing/avara: new aport

### PR DESCRIPTION
**Pull request by shrizza**

### CODE REVIEW

File: testing/avara/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+pkgver="0_git20191006"
+pkgrel=0
+pkgdesc="Port of the original 1996 Mac OS game from Ambrosia Software"
+url="https://github.com/avaraline/Avara"
+arch="all"
+license="MIT"
+depends=""
+makedepends="clang clang-dev directfb-dev glu-dev sdl2_net-dev sdl2_mixer-dev sdl2"
+options="!check"
+install=""
```
Comment (Cogitri) [inline testing/avara/APKBUILD]: no need for the empty variable here

File: testing/avara/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Contributor: shrizza <shrizza@gmail.com>
+# Maintainer: shrizza <shrizza@gmail.com>
+pkgname="avara"
+pkgver="0_git20191006"
+pkgrel=0
+pkgdesc="Port of the original 1996 Mac OS game from Ambrosia Software"
+url="https://github.com/avaraline/Avara"
+arch="all"
+license="MIT"
+depends=""
```
Comment (Cogitri) [inline testing/avara/APKBUILD]: no need for the empty variable here

File: testing/avara/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+makedepends="clang clang-dev directfb-dev glu-dev sdl2_net-dev sdl2_mixer-dev sdl2"
+options="!check"
+install=""
+subpackages="$pkgname-doc"
+source="avara https://github.com/avaraline/Avara/archive/b8700db.zip"
```
Comment (Cogitri) [inline testing/avara/APKBUILD]: Make a `_sha` variable, which you can use both here and in the builddir, so this is:
It's `$pkgname-$pkgver.zip::https://github.com/avaraline/Avara/archive/$_sha.zip`

File: testing/avara/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+builddir="$srcdir/Avara-b8700db3c696b9813b511b06a92c9faab55b282a"
+
+build() {
+	make || return 1
```
Comment (Cogitri) [inline testing/avara/APKBUILD]: No need for `|| return 1`, aports already do `set -e` (a failing command will fail the entire build by default)

File: testing/avara/avara [hunk 5]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+$( cd /var/games/avara && ./Avara )
```
Comment (Cogitri) [inline testing/avara/avara]: IMHO just do /var/games/avara/Avara instead of `cd`ing

File: testing/avara/APKBUILD [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+	cp -r build/Avara "$pkgdir/var/games/$pkgname/"
+	cp -r build/bsps "$pkgdir/var/games/$pkgname/"
+	cp -r build/levels "$pkgdir/var/games/$pkgname/"
+	cp -r build/rsrc "$pkgdir/var/games/$pkgname/"
+	cp -r build/shaders "$pkgdir/var/games/$pkgname/"
+	mkdir -p "$pkgdir/usr/share/doc/$pkgname/"
+	install -Dm644 CONTRIBUTORS.md "$pkgdir/usr/share/doc/$pkgname/"
+	install -Dm644 README.md "$pkgdir/usr/share/doc/$pkgname/"
+	mkdir -p "$pkgdir/usr/share/licenses/$pkgname"
+	install -Dm644 LICENSE.txt "$pkgdir/usr/share/licenses/$pkgname/"
```
Comment (Cogitri) [inline testing/avara/APKBUILD]: No need to install the license unless it's a custom one.

**Comment by shrizza** [comment]
Thank you for your suggestions. Hopefully the latest revision satisfies all your requirements.

**Pull request closed without merging**